src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sale = 0;
+
+  if (days >= 7) {
+    sale = 50;
+  } else if (days >= 3) {
+    sale = 20;
+  }
+
+  return days * 40 - sale;
```

Comment:
The function name `calculateRentalCost` is descriptive and clear, which is good. However, the use of literals in the expressions and the use of `let` for `sale` can be improved according to the checklist. Consider using constants for the number of days thresholds and the corresponding discounts, and use `const` for values that do not change.